llm/server.go

```diff
@@ -404,26 +428,7 @@ func NewLlamaServer(gpus discover.GpuInfoList, modelPath string, f *ggml.GGML, a
 		}
 
 		slog.Info("starting llama server", "cmd", s.cmd)
-		if envconfig.Debug() {
```

Comment:
this condition isn't necessary because `slog.Debug` won't output anything unless slog is already configured for LevelDebug or lower